src/zmalloc.c

```diff
@@ -57,6 +57,12 @@ void zlibc_free(void *ptr) {
 #endif
 #endif
 
+#if PREFIX_SIZE > 0
```

Comment:
How does this even compile if PREFIX_SIZE is defined as (sizeof(size_t)), for example?

Reply:
@ivanstosic-janea it didn't. 
Hence https://github.com/redis/redis/issues/8531